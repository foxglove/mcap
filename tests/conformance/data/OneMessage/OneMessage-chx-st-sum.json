{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {
      "type": "Schema",
      "fields": [
        ["data", ["4", "5", "6"]],
        ["encoding", "c"],
        ["id", "1"],
        ["name", "Example"]
      ]
    },
    {
      "type": "Channel",
      "fields": [
        ["id", "1"],
        ["message_encoding", "a"],
        ["metadata", {"foo": "bar"}],
        ["schema_id", "1"],
        ["topic", "example"]
      ]
    },
    {
      "type": "Message",
      "fields": [
        ["channel_id", "1"],
        ["data", ["1", "2", "3"]],
        ["log_time", "2"],
        ["publish_time", "1"],
        ["sequence", "10"]
      ]
    },
    {
      "type": "Statistics",
      "fields": [
        ["attachment_count", "0"],
        ["channel_count", "1"],
        ["channel_message_counts", {"1": "1"}],
        ["chunk_count", "0"],
        ["message_count", "1"],
        ["metadata_count", "0"]
      ]
    },
    {
      "type": "SummaryOffset",
      "fields": [
        ["group_length", "47"],
        ["group_opcode", "11"],
        ["group_start", "153"]
      ]
    },
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "0"],
        ["summary_offset_start", "200"],
        ["summary_start", "153"]
      ]
    }
  ],
  "meta": {"variant": {"features": ["st", "chx", "sum"]}}
}
