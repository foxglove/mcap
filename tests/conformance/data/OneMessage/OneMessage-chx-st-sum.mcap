version https://git-lfs.github.com/spec/v1
oid sha256:f5d2b2c74b23ae7e6bbf426b9b3cc40fbd42a0452986f9bdcdb6a75b8e5547fb
size 311
