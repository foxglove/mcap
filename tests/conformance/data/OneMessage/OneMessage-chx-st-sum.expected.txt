Header library="" profile=""
Schema id=1 schema=b schema_encoding=c schema_name=Example
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
Message channel_id=1 log_time=2 message_data=<010203> publish_time=1 sequence=10
Statistics attachment_count=0 channel_count=1 channel_message_counts={1=1} chunk_count=0 message_count=1
SummaryOffset group_length=0 group_opcode=13 group_start=194
SummaryOffset group_length=43 group_opcode=11 group_start=151
SummaryOffset group_length=0 group_opcode=10 group_start=194
SummaryOffset group_length=0 group_opcode=8 group_start=194
Footer summary_crc=0 summary_offset_start=194 summary_start=151
